Sources/SwiftReactorUIKit/ReactorView.swift

```diff
@@ -16,7 +16,7 @@ import SwiftReactor
 ///
 /// - Important: call the `setAndBind(reactor:)` method to set the reactor and call the `bind(reactor:)` method
 ///
-public protocol ReactorView: class {
+public protocol ReactorView: AnyObject {
```

Comment:
Maybe a different naming would be better here to not collide with the SwiftUI `ReactorView`. Maybe `ReactorUIView`? But then we need to raise the version accordingly.